XyzWalker/Assets/Src/Scripts/CodeExecutor.cs

```diff
@@ -47,55 +47,39 @@ private class ConsoleWriter : TextWriter {
     }
   }
 
+  public bool IsRunning => !_engine.IsStopped;
```

Comment:
Properties must go after const and readonly fields.

Reply:
Fixed.